plugins/CoreAdminHome/OptOutManager.php

```diff
@@ -172,10 +172,18 @@ public function getOptOutView()
 
         $setCookieInNewWindow = Common::getRequestVar('setCookieInNewWindow', false, 'int');
         if ($setCookieInNewWindow) {
+            $requestNonce = Common::getRequestVar('nonce', false);
+            $nonce = $requestNonce !== false ? $requestNonce : '';
```

Comment:
btw is $requestNonce actually needed here? or could in line 186 have something like `'nonce' => $nonce ? $nonce : ''`? Tried to understand and was bit complicated.

Reply:
I'll check it, but probably it's not needed like this anymore. I'll update the pr.

Reply:
updated